src/Simulation.cpp

```diff
@@ -313,6 +313,7 @@ void Simulation::readXML(XMLfileUnits& xmlconfig) {
 				_domainDecomposition = new KDDecomposition(getcutoffRadius(), _ensemble->getComponents()->size());
 			} else if (parallelisationtype == "GeneralDomainDecomposition") {
 				double skin = 0.;
+				bool forceGrid = false;
```

Comment:
The name is highly misleading. The domain decomposition is always some form of a grid. I guess you mean something like "force to fixed (regular?) grid"

Reply:
renamed to `forceLatchingToRegularGrid`